frame/contracts/src/wasm/runtime.rs

```diff
@@ -269,6 +269,8 @@ pub enum RuntimeCosts {
 	/// Weight of calling `account_reentrance_count`
 	#[cfg(feature = "unstable-interface")]
 	AccountEntranceCount,
+	/// Weight of calling `instantiation_nonce`
```

Comment:
Shouldn't this be gated as well?

```suggestion
	/// Weight of calling `instantiation_nonce`
	#[cfg(feature = "unstable-interface")]
```